### PR TITLE
Add pre release process

### PR DESCRIPTION
**Pull request by bidoubiwa**

### CODE REVIEW

File: playgrounds/angular/src/app/app.component.ts [hunk 1]
```diff
@@ -2,8 +2,10 @@ import { Component } from '@angular/core'
 import { instantMeiliSearch } from '../../../../src'
 
 const searchClient = instantMeiliSearch(
-  'https://demos.meilisearch.com',
-  'dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25'
+  // 'https://demos.meilisearch.com',
+  // 'dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25'
+  'http://localhost:7700',
+  'masterKey'
```
Comment (bidoubiwa) [inline playgrounds/angular/src/app/app.component.ts:8]:
```suggestion
  'https://demos.meilisearch.com',
  'dc3fedaf922de8937fdea01f0a7d59557f1fd31832cb8440ce94231cfdde7f25'
```

**Review by curquiza — DISMISSED** [review]

**Review by curquiza — APPROVED** [review]

**Review by curquiza — APPROVED** [review]
👍

**Comment by curquiza** [comment]
Oops, the types-check are not working, I approved before seeing this

**Comment by curquiza** [comment]
checked with @bidoubiwa we merge this PR despite the type-check failing

**Pull request merged**